src/app/helptext/storage/volumes/datasets/dataset-permissions.ts

```diff
@@ -1,38 +1,41 @@
 import { T } from '../../../../translate-marker';
 
 export default {
-dataset_permissions_mp_path_placeholder: T('Path'),
+dataset_permissions_id_placeholder: T('Path'),
 
-dataset_permissions_mp_acl_placeholder: T('ACL Type'),
-dataset_permissions_mp_acl_tooltip: T('Select the type that matches the type of client\
+dataset_permissions_acl_placeholder: T('ACL Type'),
+dataset_permissions_acl_tooltip: T('Select the type that matches the type of client\
  accessing the dataset.'),
 
-dataset_permissions_mp_user_en_placeholder: T('Apply User'),
-dataset_permissions_mp_user_en_tooltip: T('Apply changes to the user.'),
+dataset_permissions_user_en_placeholder: T('Apply User'),
```

Comment:
Not clear what "_en_" means here and in some later fields.  I'd suspect it means English, but that's not usually how we do things.  They are not used in or are removed from the other files in this diff.  Are they duplicates that can be removed?

Reply:
thanks @wblock. They should be removed.